class-3-branches-reset/commands.txt

```diff
@@ -1,6 +1,9 @@
 git remote add <alias_name> <.git url>
 git remote rm <alias_name>
+git remote -v 
 git branch
 git checkout -b <branch_name>
 git checkout <branch_name>
-git switch <branch_name>
\ No newline at end of file
+git switch <branch_name>
+git reset <commit_id>  --hard/soft/mixed 
```

Comment:
Hey @vikash123765 , the commands seem correct but a small suggestion that -- needs to be in all the modes and not just in hard
Fix :  git reset <commit_id> --hard/--soft/--mixed

also note that u dont necessarily need the commit id, U may also use the head pointer of a branch
Fix : git reset head~1 --hard/--soft/--mixed